publish-package.sh

```diff
@@ -5,20 +5,20 @@ set -e
 CURRENT_BRANCH=`git name-rev --name-only HEAD`
 
 if [ $CURRENT_BRANCH != 'master' ] ; then
```

Comment:
It has been like this the whole time, so maybe that was an oversight of @pfiller; people can open a pull request from their master branch. Or don't these PR's trigger a build?